azurelinuxagent/ga/exthandlers.py

```diff
@@ -67,6 +67,7 @@
 
 TELEMETRY_MESSAGE_MAX_LEN = 3200
 
+DEFAULT_EXT_TIMEOUT_MINUTES = 90
```

Comment:
It looks like this is a per-extension timeout, won't CRP timeout well before this for multiple extensions?

Reply:
This is the timeout value per dependency level.
CRP has an independent time out value. I will start an email thread including the CRP-side owner, on when/how to update the CRP timeout value.